src/agor/strategy.py

```
@@ -234,34 +233,23 @@ def _create_agent_branches(self, task: str, agents: int) -> Dict[str, str]:
-                )
-            except Exception:
-                pass  # Branch might already exist
+            # Using run_git_command. Assuming project_root is the correct cwd.
+            # run_git_command doesn't have check=False, it returns (success, output).
+            # We want to create if not exists, so if it fails because it exists, that's okay.
+            # If it fails for other reasons, that might be an issue.
+            # For now, mimic check=False by not strictly checking `success`.
+            run_git_command(
+                ["checkout", "-b", branch_name],
+                cwd=str(self.project_root) # run_git_command might need cwd as string
+            )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'cwd' in function call

(E1123)